decisions/2025-06-23_excel_reports.md

```diff
@@ -0,0 +1,104 @@
+# Exporting Excel Reports from Open mSupply
+
+- _Date_: 2025-06-23
+- _Deciders_: @lache-melvin, @andreievg
+- _Status_: DECIDED
+- _Outcome_: Option 3 - HTML Attributes + optional Excel template
+
+## Context
+
+OMS Reports (and forms) are currently generated from an HTML template, by populating the template with some data. These can then be printed (by printing the rendered HTML page) or converted to Excel.
+
+At the time of writing, export to Excel only takes the inner data table of the report (the header row and the data rows), and renders these rows in an Excel table.
+
+This is a great start, but fails to support more complex needs:
```

Comment:
Maybe 'advanced requirements', 'demanding need' maybe, although those lead to complexity, that's not the goal haha, just a nit: